.ci/build.sh

```diff
@@ -85,6 +92,7 @@ for chart in $charts; do
   any_chart_changed=true
 
   echo "----> Build dependencies for ${chart}"
+  helm dependency update
```

Comment:
I don't think this will work as expected. The `dependency update` command it used to update the `requirements.lock` file and to actually update dependencies. So this shouldn't run in the pipeline. Running `dependency build` should be enough.

Reply:
There is an open issue regarding this: https://github.com/helm/helm/issues/7214